src/main.ts

```diff
@@ -7,26 +7,23 @@ import router from './router'
 // Import stylesheet
 import 'vue-loading-overlay/dist/vue-loading.css'
 import 'bootstrap'
-import fetchPonyfill from 'fetch-ponyfill'
 
 Vue.config.productionTip = false
 
-// Work around for session key time out
-fetchPonyfill().fetch('/api/v2/i18n/lifelines-webshop/en').then((resp) => {
-  if (resp.status === 401) {
-    window.location.href = '/login'
-  }
-})
-
-Vue.use(i18n, {
-  lng: 'en',
-  fallbackLng: 'en',
-  namespace: ['lifelines-webshop', 'ui-form'],
-  callback () {
-    new Vue({
-      store,
-      router,
-      render: h => h(App)
-    }).$mount('#app')
-  }
-})
+store.dispatch('fetchContext').then(() => {
+  Vue.use(i18n, {
```

Comment:
chaining the context and i18n load will bad first paint time even worse

Reply:
This refactor changes the initial "is the session okay" call from one to the i18n endpoint to one on the context endpoint.

Reply:
done